sub-projects/hub/src/app/components/home/home-page.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+import {
+  Button,
+  Segment,
+  Divider,
+  Header,
+} from 'semantic-ui-react';
+
+export default class HomePage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      navigateToAddAdmin: false,
+      navigatetoAddPOI: false,
+      navigatetoViewPOI: false,
+    };
+  }
+
+  render() {
```

Comment:
I'm not going to ask for a change, but couldn't you just create a function that pushes to the history instead of dealing with a re-render?